src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2738,7 +2739,7 @@ private bool IsMisplacedModifier(SyntaxListBuilder modifiers, SyntaxList<Attribu
                 }
 
                 var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
-                if (incompleteMember.ContainsDiagnostics)
+                if (ContainsErrorDiagnostic(incompleteMember))
```

Comment:
I didn't understand how this change relates to the change in the PR. Can you help?

Reply:
the check for .ContainsDiagnostics is incorrect.  That bit is true if there are *any* diagnostics on or under the node.  THat includes *warnings* (like the lexer `l`-warning).  So in the presence of a warning, we don't report an actual error.  

The intent of this check was "we've already reported an error, no need to report another one".  The new logic actually provides that concept.

Reply:
Thanks